<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hierarchy</class>
 <widget class="QWidget" name="Hierarchy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeHierarchy">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>SCENE</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
      <property name="background">
       <color>
        <red>103</red>
        <green>103</green>
        <blue>102</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="icon">
       <iconset resource="../ShapePainterApp/icons/Icons.qrc">
        <normaloff>:/icons/Picture.png</normaloff>:/icons/Picture.png</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../ShapePainterApp/icons/Icons.qrc">
         <normaloff>:/icons/Orb plus.png</normaloff>:/icons/Orb plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../ShapePainterApp/icons/Icons.qrc">
         <normaloff>:/icons/Orb minus.png</normaloff>:/icons/Orb minus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ShapePainterApp/icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
